impl/src/testing.rs

```diff
@@ -24,6 +41,29 @@ dependencies = [
     "
 }
 
+pub fn named_toml(name: &str, version: &str) -> String {
```

Comment:
Would prefer slightly more verbose names for these two.

"named_toml_contents" "named_lock_contents".